src/components/ClusterNodes/ClusterDetails.tsx

```diff
@@ -553,7 +553,10 @@ export default function ClusterDetails({ imageList, isSuperAdmin, namespaceList,
                 <Tippy className="default-tt" arrow={false} placement="top" content={column.label}>
                     <span className="dc__inline-block dc__ellipsis-right mw-85px ">{column.label}</span>
                 </Tippy>
-                {column.isSortingAllowed && <Sort className="pointer icon-dim-14 dc__position-rel sort-icon" />}
+                {column.isSortingAllowed ? sortByColumn.value === column.value ? (
+                    <span className={`sort-icon ${sortOrder == OrderBy.DESC && 'desc' } ml-4`}></span>
```

Comment:
sortOrder == OrderBy.DESC ? 'desc' : '' 
use this instead as it will print false or undefined when condition get falsed

Reply:
done